ci/install.sh

```diff
@@ -28,9 +35,14 @@ do
       -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -c -o standby_slots.o /aiven-extras/src/standby_slots.c
     gcc -fPIC -shared -o aiven_extras.so standby_slots.o -L/usr/lib/postgresql/${dest} \
       -L/usr/lib64  -L/usr/lib64 -Wl,--as-needed -Wl,-rpath,/usr/lib/postgresql/${dest},--enable-new-dtags
+
     mkdir -p /usr/lib/postgresql/${dest}/lib/
     cp aiven_extras.so /usr/lib/postgresql/${dest}/lib/
     cp /aiven-extras/aiven_extras.control /usr/share/postgresql/${dest}/extension/
     cp /aiven-extras/sql/*.sql /usr/share/postgresql/${dest}/extension/
-done
 
+    make -C /aiven-pg-security PG_CONFIG="/usr/lib/postgresql/${dest}/bin/pg_config" clean install
+
+    # Count we have 2 entries: aiven_gatekeeper.so and aiven_extras.so.
+    $(ls /usr/lib/postgresql/${dest}/lib/aiven_{gatekeeper,extras}.so | wc -l | grep -q 2) || exit 1
```

Comment:
nit: why is it so strict that we need to check for 2 lines exactly? nothing serious but will need adjustments in case of symlinks whatnot will be added